frontend/src/app/assess/[projectId]/fairness-bias/options/page.tsx

```
@@ -206,6 +265,72 @@ export default function FairnessBiasOptions() {
           </div>
+
+          {/* Recent Evaluations Section */}
+          {isPremium && (
+            <div className="max-w-5xl mx-auto animate-in fade-in slide-in-from-bottom-4 duration-500 delay-300">
+              <div className="flex items-center justify-between mb-6">
+                <h2 className="text-xl font-semibold text-foreground">Recent Evaluations</h2>
+                {recentReports.length > 0 && (
+                  <button
+                    onClick={() => router.push(`/assess/${projectId}/fairness-bias/dataset-testing`)}
+                    className="text-sm font-medium text-primary hover:text-primary/80 transition-colors"
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)